heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -121,14 +121,14 @@ function process_message()
     msg.Fields.appUpdateChannel = read_message("Fields[appUpdateChannel]")
     msg.Fields.appBuildId       = read_message("Fields[appBuildId]")
 
+    local info = parsed.info
```

Comment:
`info` is not a top level object in the new telemetry format it is actually `parsed.payload.info`.  This will reject all new messages.